.github/workflows/devbuild.yml

```diff
@@ -177,6 +177,17 @@ jobs:
           CMAKE_ARGS="-DPYTHON_EXECUTABLE=$(which python3)"
         make buildext VERBOSE=1
 
+    # build with pytest helper binding for testing
```

Comment:
we first build the version without the helper, then build again with the helper, since later on we will run pytest.